.github/workflows/failureNotifications.yml

```diff
@@ -1,13 +1,12 @@
 name: failureNotifications
-
 on:
   workflow_run:
     workflows:
       - version, tag and github release
```

Comment:
why keep this if you deleted that workflow

Reply:
Cause i forgot 😅